src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1701,13 +1882,17 @@ bool isRefEscape
             ImmutableArray<RefKind> argRefKindsOpt,
             ImmutableArray<ParameterSymbol> parameters,
             ImmutableArray<int> argsToParamsOpt,
-            ref ArrayBuilder<bool> inParametersMatchedWithArgs)
+            ref ArrayBuilder<bool>? inParametersMatchedWithArgs,
+            out DeclarationScope scope)
         {
+            int paramIndex = argIndex < parameters.Length ?
```

Comment:
we had a local function helper elsewhere in the file. There's a similar helper also in NullableWalker. Can we check if there's a single home for this helper which would be appropriate?

Reply:
I didn't find a matching helper. In this file, there's `getParameter()` in the method above, but that returns a `ParameterSymbol` rather than an index. In `NullableWalker`, there's `getArgumentOrdinalFromParameterOrdinal()` which does the reverse mapping. Let me know if I'm overlooking something.